_platforms/it/otello.md

```diff
@@ -14,7 +14,9 @@ layout: platform
 
 ## Intro
 
-OTELLO è la piattaforma dell'[Agenzia delle Dogane e dei Monopoli](https://www.agenziadoganemonopoli.gov.it/) che gestisce il processo che consente ai soggetti extra-UE di ottenere lo sgravio diretto o il rimborso dell'IVA sui beni acquistati sul territorio italiano.
+OTELLO è la piattaforma dell'[Agenzia delle Dogane e dei Monopoli](https://www.agenziadoganemonopoli.gov.it/){:target="_blank"} che gestisce il processo che consente ai soggetti extra-UE di ottenere lo sgravio diretto o il rimborso dell'IVA sui beni acquistati sul territorio italiano.
+
+<iframe title="Otello Tax free shopping – Italiano" width="560" height="315" src="https://www.youtube.com/embed/ejVjgkFUPxo" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" class="d-block mx-auto mb-2" allowfullscreen></iframe>
```

Comment:
For privacy policy matters I think it's better to just add a link to the page where the video stays (if there's an embed somewhere on the official pages even better). Also, directly embedding a video can result in accessibility issues. WDYT?

Reply:
@libremente fixed with an internal image